pkg/network/usm/kafka_monitor_test.go

```diff
@@ -100,6 +101,18 @@ type groupInfo struct {
 	msgs    []Message
 }
 
+// isUnsupportedUbuntu checks if the test is running on an unsupported Ubuntu version.
+// As of now, we don’t support Kafka TLS with Ubuntu 24.10, so this function identifies
+// if the current platform and version match this unsupported configuration.
```

Comment:
didn't you say the issue is only on x64?

Reply:
Correct, I'll fix that